src/gt4py/next/iterator/ir.py

```diff
@@ -186,6 +186,10 @@ def _output_validator(self: datamodels.DataModelTP, attribute: datamodels.Attrib
     "cast_",
 }
 
+GTIR_BUILTINS = {
+    "apply_stencil",  # `apply_stencil(stencil)` creates field_operator from stencil
```

Comment:
I prefer the curried version, because it expresses taking a stencil and making it a field_operator.
But `apply_stencil` is a wrong name for the curried version, therefore I propose `flift` and happy to hear a better name.

Reply:
How about `as_field_operator`? `flift` feels really ugly for a PR that is supposed to be a step of a cleanup.

Reply:
I prefer the shorter name. I don't think there is a good verbose name, the `as_field_operator` should actually be `stencil_as_field_operator` unless we want overloads for `as_field_operator`, in case there are other things that could be promoted to field_operators (e.g. scalar_operators).
I'll change to `as_field_operator` for now, because I don't see us figuring out a good name easily.